Scripts/script-CommonServer.yml

```diff
@@ -534,6 +534,20 @@ script: |-
       return res;
   }
 
+  /**
+   * Get list data which is comma-separated into an array
+   * @param {String} listName - the name of the list
+   * @returns {Array} the list as an array
+   */
+  function getCSVListAsArray(listName) {
```

Comment:
add an optional format parameter,
Since lists maybe managed via scripts and then be in proper json format